orttraining/orttraining/core/session/training_session.cc

```diff
@@ -166,15 +212,21 @@ Status TrainingSession::ConfigureForTraining(
                                          config.distributed_config.horizontal_parallel_size,
                                          config.distributed_config.pipeline_parallel_size});
 
-  const int32_t pipeline_stage_id = config.pipeline_config.has_value() ?
-                              DistributedRunContext::RankInGroup(WorkerGroupType::ModelParallel) :
-                              -1;
+  const int32_t pipeline_stage_id = config.pipeline_config.has_value() ? DistributedRunContext::RankInGroup(WorkerGroupType::ModelParallel) : -1;
```

Comment:
Should we rename WorkerGroupType::ModelParallel -> WorkerGroupType::PipelineParallel?  It will avoid the ambiguity over whether "Model" means pipeline-style or megatron-style.

Reply:
Sure.

---
In reply to: [527648455](https://github.com/microsoft/onnxruntime/pull/5815#discussion_r527648455) [](ancestors = 527648455)